### ISSUE TITLE
Add option to chenge .history folde name 

### ISSUE THREAD

**Issue by MarekLew** [zabel-xyz/local-history#62]
I have problem when one workspace is inside another:
My idea of structure is:
```
- parent-folder
    - .history
        - workspace 1
            - workspace 2  <--- this is .history from workspace2
    - workspace 1
        - .vscode
        - workspace 2
            - .vscode  
```
At the moment a use symlink :
`parent-folder/.history/workspace 1/workspace 2 --> parent-folder/workspace 1/workspace 2/.history`

But if you could change the name for `.history`, I could set a direct path.

**Comment by zabel-xyz** [comment]
There is already a PR like your request. See #54
But I must analyse and see all impacts of such a change, before to implement such a feature...

**Comment by MarekLew** [comment]
Yes #54 is not complete. This is more 3-4 please to fixed. This is not that simply like #54.
